package.json

```diff
@@ -38,7 +45,7 @@
         "ts-node": "^10.9.1",
         "typescript": "^5.0.4"
     },
-    "packageManager": "yarn@3.6.3",
+    "packageManager": "yarn@3.6.4",
```

Comment:
Does this actually bump anything. `.yarnrc.yml` still points towards 3.6.3

Reply:
Doc says:

> Corepack will make sure that all calls to the relevant binaries are run against the requested version, downloading it on demand if needed, and aborting if it cannot be successfully retrieved.